tests/conftest.py

```diff
@@ -135,6 +135,16 @@ def cfg():
     return Config()
 
 
+@pytest.fixture
+def cfg_override():
+    class ConfigOverride(Config):
```

Comment:
Please could we add something to the documentation, perhaps an example, showing this approach for overriding the config object?